airflow/providers/facebook/ads/hooks/ads.py

```diff
@@ -105,24 +110,74 @@ def bulk_facebook_report(
         params: Dict[str, Any],
         fields: List[str],
         sleep_time: int = 5,
-    ) -> List[AdsInsights]:
+    ) -> Union[List[AdsInsights], Dict[str, List[AdsInsights]]]:
         """
-        Pulls data from the Facebook Ads API
+        Pulls data from the Facebook Ads API regarding Account ID with matching return type.
+        The return type and value depends on the ``account_id`` configuration. If the
+        configuration is a str representing a single Account ID, the return value is the
+        list of reports for that ID. If the configuration is a list of str representing
+        multiple Account IDs, the return value is a dict of Account IDs and their
+        respective list of reports.
```

Comment:
```suggestion
        """Pulls data from the Facebook Ads API regarding Account ID with matching return type.
        
        The return type and value depends on the ``account_id`` configuration. If the
        configuration is a str representing a single Account ID, the return value is the
        list of reports for that ID. If the configuration is a list of str representing
        multiple Account IDs, the return value is a dict of Account IDs and their
        respective list of reports.
```

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

https://www.python.org/dev/peps/pep-0257/

Reply:
Hi @uranusjr, sorry for the late reply. I have made the changes. Thank you,